wishlist.css

```diff
@@ -0,0 +1,15 @@
+.wishlist-section {
+  padding: 3rem;
+  display: grid;
+  column-gap: 20px;
+  /* column gap not respected why? */
```

Comment:
I think you should remove this comment 😅

Reply:
okay it was for self reference xD